core/client/src/main/java/alluxio/hadoop/HdfsFileInputStream.java

```diff
@@ -390,4 +390,18 @@ public void seek(long pos) throws IOException {
   public boolean seekToNewSource(long targetPos) throws IOException {
     throw new IOException(ExceptionMessage.NOT_SUPPORTED.getMessage());
   }
+
+  @Override
```

Comment:
OK，got it! Thanks for the comment! I may finish it tomorrow because I have some other work to do on Saturday!